source/HiringPanel.cpp

```diff
@@ -106,11 +106,20 @@ bool HiringPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, b
 	if(key == 'h' || key == SDLK_EQUALS || key == SDLK_KP_PLUS || key == SDLK_PLUS || key == SDLK_RETURN || key == SDLK_SPACE)
 	{
 		player.Flagship()->AddCrew(min(maxHire, Modifier()));
+		player.Flagship()->SetDesiredCrew(player.Flagship()->Crew());
 		player.UpdateCargoCapacities();
 	}
 	else if(key == 'f' || key == SDLK_MINUS || key == SDLK_KP_MINUS || key == SDLK_BACKSPACE || key == SDLK_DELETE)
 	{
 		player.Flagship()->AddCrew(-min(maxFire, Modifier()));
+		// If the ship has more than the minimum required crew, then set the desired crew
+		// to the value the player requested. If the ship has the exact amount (or even
+		// less), then the planer apparently doesn't want to hire any extra crew (and
+		// thus we set the desired crew value to zero).
```

Comment:
This is overly verbose for the code that follows. Comments need not say what the code will do; they need to provide the context of the _why_.

(I think resolving the other feedback will probably remove the need for this comment at all.)